config.go

```diff
@@ -0,0 +1,145 @@
+package main
+
+import (
+	"fmt"
+	"os"
+	"path/filepath"
+	"strings"
+
+	"github.com/btcsuite/go-flags"
+	"github.com/decred/dcrd/dcrutil"
+)
+
+const (
+	defaultTLSCertFilename  = "tls.cert"
+	defaultMacaroonFilename = "admin.macaroon"
+	defaultLogFilename      = "dcrlnfaucet.log"
+	defaultConfigFilename   = "dcrlnfaucet.conf"
+	defaultLogLevel         = "info"
+	defaultLndIP            = "10.0.0.9"
+	defaultNetParams        = "testnet"
+	defaultLndNodes         = "localhost:10009"
+	defaultBindAddr         = ":80"
+	defaultUseLeHTTPS       = false
+	defaultWipeChannels     = false
+	defaultDomain           = "faucet.lightning.community"
```

Comment:
It's probably better to _not_ specify a default for the domain and check whether it _is_ specified when `-use_le_https` is used.